docs/api/go/examples/transfer.md

```diff
@@ -112,7 +112,7 @@ func main() {
 	var (
 		PrivateKey1       = os.Getenv("PRIVATE_KEY")
 		PublicKey2        = "0x81E9D85b65E9CC8618D85A1110e4b1DF63fA30d9"
-		ZkSyncEraProvider = "https://testnet.era.zksync.dev"
+		ZkSyncEraProvider = "https://sepolia.era.zksync.dev"
 		TokenL2Address    = common.HexToAddress("0xCd9BDa1d0FC539043D4C80103bdF4f9cb108931B")
```

Comment:
0xCd9BDa1d0FC539043D4C80103bdF4f9cb108931B does not exist on Sepolia

Reply:
We'll be reverting the Go SDK examples/docs back to use Goerli Testnet until we can find an appropriate path forward. Good catch!